openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -121,7 +121,7 @@ defined in [@ISO.18013-7]. For more details, also see Annex B.3 in [@!OIDF.OID4V
 Both the Wallet and the Credential Issuer:
 
 * MUST support the authorization code flow.
-* MUST support protocol extensions for the SD-JWT VC Credential format profile as defined in (#vc_sd_jwt_profile).
+* MUST support at least one of the Credential format profiles and protocol extensions defined in (#vc_profiles).
```

Comment:
Should we add some note that an ecosystem should make sure to guarantee interop by either choosing one format or mandating support for both? The statement as will imho likely cause confusion or interop problems.